projects/angular/src/forms/datepicker/datepicker-view-manager.ts

```diff
@@ -16,11 +17,12 @@ import { ViewManagerService } from './providers/view-manager.service';
   host: {
     '[class.datepicker]': 'true',
     '[attr.aria-modal]': 'true',
+    '[attr.aria-label]': 'commonStrings.keys.datepickerDialogLabel',
     role: 'dialog',
   },
 })
 export class ClrDatepickerViewManager {
-  constructor(private viewManagerService: ViewManagerService) {}
+  constructor(public commonStrings: ClrCommonStringsService, private viewManagerService: ViewManagerService) {}
```

Comment:
Changes are fine but the backport's build is failing bad. I'm going to block and try to backport locally and see if I can resolve these build issues.